### PR TITLE
Single folder

### PR DESCRIPTION
**Pull request by derekdowling**
Moving us into a single folder organization. Makes way more sense and is easier to navigate for whoever might integrate with us. Only moved content_api in for now since there is a bunch of working being done on the author side right now. I will do that after just to ensure we don't run into any issues.

### CODE REVIEW

File: api/tests/test_content_api.py [hunk 1]
```diff
@@ -68,12 +67,12 @@ def tearDown(self):
...
         try:
-            user = User.objects.get(username = USER_A["username"])
-            user = User.objects.get(id = self.user_a.id)
+            user = User.objects.get(username=USER_A["username"])
+            User.objects.get(id=self.user_a.id)
         except:
             self.assertFalse(True, "Error retrieving %s from database" %USER_A["username"])
         try:
-            post = Post.objects.get(guid=self.post.guid)
+            Post.objects.get(guid=self.post.guid)
```
Comment (lanre-ade) [inline api/tests/test_content_api.py:75]: why did you take out the post assignment?

Comment (derekdowling) [inline api/tests/test_content_api.py:75]: Because it's not being used anywhere. We're really just testing if it throw an exception.

**Comment by derekdowling** [comment]
All the tests are confirmed to be passing as well.

File: api/tests/test_content_api.py (hunk 1 above)
Comment (lanre-ade) [inline api/tests/test_content_api.py:75]: Ah I just realized we're using self.post afterwards, fair enough

**Pull request merged**